Array/readme.md

```diff
@@ -12,19 +12,22 @@ The syntax is:
 
 ### Advantages
 
-* Random access of elements using array index.
+- Random access of elements using array index.
 
-* Use of less line of code as it creates a single array of multiple elements.
+- Use of less line of code as it creates a single array of multiple elements.
 
-* Easy access to all the elements.
+- Easy access to all the elements.
 
 ### Disadvantages
 
 To create an array, you need to known the length (or size) of the array in advance, and allocate accordingly. Once an array is created, its length is fixed and cannot be changed. At times, it is hard to ascertain the length of an array. Nonetheless, you need to estimate the length and allocate an upper bound. This is probably the major drawback of using an array.
- 
+
 ## Questions :
 
- * Insertion_at_start ----> [C++](/Code/C++/insertion_at_start.cpp) 
- * Kadane's Algorithm ----> [C++](/Code/C++/kadane_algo.cpp) 
- * Repeating and Missing Number ----> [C++](/Code/C++/repeating_and_missing_number.cpp) 
+- Insertion_at_start ----> [C++](/Code/C++/insertion_at_start.cpp)
+- Kadane's Algorithm ----> [C++](/Code/C++/kadane_algo.cpp)
+- Repeating and Missing Number ----> [C++](/Code/C++/repeating_and_missing_number.cpp)
 
+* Insertion_at_start ----> [C++](/Code/C++/insertion_at_start.cpp) | [Java](/Code/Java/insertion_at_start.java) | [Python](/Code/Python/insertion_at_start.py)
+* Kadane's Algorithm ----> [C++](/Code/C++/kadane_algo.cpp)
```

Comment:
Index only those codes that this PR is supposed to add.

Reply:
The VS Code's inbuild Markdown formatter does that changes to improve the spaces in all the lines for a uniform format through out the Readme file. The added line is just one , rest formatting is done by VS code just to make it uniform.